keps/sig-scheduling/5027-dra-admin-controlled-device-attributes/README.md

```diff
@@ -166,20 +166,8 @@ scheduler when selecting devices for user requests in ResourceClaims.
 This KEP adds a Kubernetes API that privileged users, typically cluster
 administrators, can use to override or extend that information. This can be
```

Comment:
can the admin be a maintenance controller? if so can we add the same to the KEP

Reply:
Yes, will add.